src/screenplay.py

```diff
@@ -1037,7 +1021,6 @@ def generatePMLPage(self, pager, pageNr, forPDF, doExtra):
                         s = text
 
                     to.toc = pml.TOCItem(s, to)
-                    pager.doc.addTOC(to.toc)
```

Comment:
Adding all table of contents items to the document is not necessary any more – it is now enough to have them on the text objects that should appear in the table of contents.